frontend/src/app/components/api-docs/api-docs.component.ts

```diff
@@ -2417,7 +3294,7 @@ responseSample: `{
 
         const currencies = await markets.getCurrencies();
         console.log(currencies);`,
-        curl: `curl -X GET "https://bisq.markets/api/markets/currencies/"`,
+        curl: `curl "https://bisq.markets/api/currencies/"`,
```

Comment:
```suggestion
        curl: `curl "https://bisq.markets/api/currencies"`,
```

Reply:
I am right in Franklin the rifle owner of these transactions I'm trying to gain my full access keep fraud off my case